nixos/modules/services/web-servers/nginx/vhost-options.nix

```diff
@@ -221,7 +222,7 @@ with lib;
         IP address / port.
         If there is one server block configured to enable http2, then it is
         enabled for all server blocks on this IP.
-        See https://stackoverflow.com/a/39466948/263061.
+        See <https://stackoverflow.com/a/39466948/263061>.
```

Comment:
Stackoverflow as documentation resource? I am impressed

Reply:
Ah ops, I added brackes on it as suggested, but on the wrong one, lol